src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  const calculate = price * days;
+
+  if (days >= 7) {
+    return calculate - 50;
+  } else if (days >= 3) {
+    return calculate - 20;
+  }
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 3rd item, you don't need `else` here